kitchen.yml

```diff
@@ -11,9 +11,7 @@ verifier:
   name: inspec
 
 platforms:
-  - name: amazon-linux
-    driver_config:
-      box: mvbcoding/awslinux
+  - name: amazonlinux-2
```

Comment:
I think we should probably vote on this one in the next meeting. We can probably look at removing Amazon-linux testing and replace it with amazonlinux-2 testing 👍

Reply:
That would help reduce the amount of testing scenarios